doc/7.Measurements.md

```diff
@@ -277,6 +277,8 @@ Assertion 7.5.2:
 Assertion 7.5.3:
     if (SpdmMeasurementBlock[i].Measurement.DMTFSpecMeasurementValueType.bit_7 = 0) then (SpdmMeasurementBlock[i].Measurement.DMTFSpecMeasurementValueSize == HashSize)
 
+Assertion 7.5.4:
+    When DMTFSpecMeasurementValueType[6:0]=0x5 , DMTFSpecMeasurementValueType[7] shall be set to 1b .

```

Comment:
1) Please leave a blank line. Follow the same style.

2) Please use the same style for description, such as AAA == BBB, if CCC, or if () then (AAA == BBB)